src/DonorProfiles/Repositories/Donations.php

```diff
@@ -193,15 +193,65 @@ protected function getFormInfo( $payment ) {
 	 * @return array Payment info
 	 */
 	protected function getPaymentInfo( $payment ) {
+
+		$gateways = give_get_payment_gateways();
+
 		return [
-			'amount'   => $payment->subtotal,
+			'amount'   => $this->getFormattedAmount( $payment->subtotal, $payment ),
 			'currency' => $payment->currency,
-			'fee'      => ( $payment->total - $payment->subtotal ),
-			'total'    => $payment->total,
-			'method'   => $payment->gateway,
-			'status'   => $payment->status,
-			'date'     => $payment->date,
+			'fee'      => $this->getFormattedAmount( ( $payment->total - $payment->subtotal ), $payment ),
+			'total'    => $this->getFormattedAmount( $payment->total, $payment ),
+			'method'   => $gateways[ $payment->gateway ]['checkout_label'],
+			'status'   => $this->getFormattedStatus( $payment->status ),
+			'date'     => date( 'F j, Y', strtotime( $payment->date ) ),
```

Comment:
@henryholtgeerts Is the time format hardcoded on the donor profile page? in the core, we show the time format as per the website setting.

![image](https://user-images.githubusercontent.com/1784821/107025109-f4a04480-67ce-11eb-8e8c-d0f865dd7b2c.png)

Reply:
Ah - good suggestion! I was working off the Figma designs, but this is important to consider. Done, ref: https://github.com/impress-org/givewp/pull/5566/commits/782f2a24c783dd7dc8b76cc848dc6e1d8ddfc008